client/src/components/Button/DoubleActionButton.jsx

```diff
@@ -0,0 +1,12 @@
+import './DoubleActionButton.scss';
```

Comment:
My understanding of the double action button is that essentially we want two buttons next to each other, each with its own `onClick` and text

Reply:
Thanks for the feedback, Nick! This all makes sense. Based on my understanding I'll:

- change Links to buttons with onClick and type attributes
- onClick function, text, and type will be passed into component as props

Will work on fixing these now 👍

Reply:
Ooh, interesting. Okay I've updated per notes above. Will consider adapting further for buttons not needing onClicks